src/main/java/com/minecolonies/coremod/colony/managers/BuildingManager.java

```diff
@@ -26,9 +26,7 @@
 
 import java.util.*;
 
-import static com.minecolonies.api.util.constant.NbtTagConstants.TAG_BUILDINGS;
-import static com.minecolonies.api.util.constant.NbtTagConstants.TAG_NEW_FIELDS;
-import static com.minecolonies.api.util.constant.NbtTagConstants.TAG_POS;
+import static com.minecolonies.api.util.constant.NbtTagConstants.*;
 
 public class BuildingManager implements IBuildingManager
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1258)